### ISSUE TITLE
Cannot open or edit .drawio.svg files

### ISSUE THREAD

**Issue by deadlydog** [hediet/vscode-drawio#124]
I'm able to create and edit .drawio and .drawio.svg files fine, but when I convert them to a .drawio.svg file and then try to open that in the editor, VS Code gives the error:

> Unable to open 'test.drawio.svg': File seems to be binary and cannot be opened as text.

I tried updating my settings.json to explicitly tell it to use the editor by adding:

```json
    "workbench.editorAssociations": [
        {
            "viewType": "hediet.vscode-drawio-text",
            "filenamePattern": "*.drawio.svg"
        }
    ]
```

but still get the same error. It seems the file extension is not being associated to your VS Extension editor for some reason.

Any ideas on how to fix it? My environment info is below. Thanks!

VS Code Info:
```
Version: 1.48.2 (user setup)
Commit: a0479759d6e9ea56afa657e454193f72aef85bd0
Date: 2020-08-25T10:13:11.295Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362
```

Draw.io Integration v0.7.2

**Comment by hediet** [comment]
Thanks for your report ;) Can you describe a repro in more detail (I was not able to reproduce it)? Can you try to use the insider build of the drawio extension?
Thank you!

**Comment by deadlydog** [comment]
Thanks for the quick response :) 

I installed the Insider Build extension and then was able to convert to svg and load it without issue. I then uninstalled the Insider Build and installed the regular extension again, and I'm still able to load svg files without issue.

I hadn't tried reinstalling the extension before reporting the issue, so I'm not sure if it was simply the reinstall that fixed it, or if something about installing the Insider Build fixed it ¯\_(ツ)_/¯

But everything does appear to be working correctly now. Thanks!
